CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+<img src="services/site/public/images/a11yphant_Logo_Signet.png" height="138">
+
+# Contributing to a11yphant
+
+Looking to contribute something to a11yphant? **Here's how you can help.**
+
+
+<hr style="color:white; height:1.5px; width:100%;"/>
+
+## Table of Contents
+
+- [Documentation](DOCUMENTATION.md)
+- [Reporting Bugs and Issues](#reporting-bugs-and-issues)
+- [Feature Requests](#feature-requests)
+    - [Create a Pull Request](#create-a-pull-request)
+- [Writing Content](#writing-content)
+
+<hr style="color:white; height:1.5px; width:100%;"/>
+
+We decided to continue a11yphant as Open Source project. Our goal is to make the web a more accessible and inclusive place. We do not want to make profit. All money we might receive through sponsorships goes directly into developing a11yphant further.
+
+## Read the Documentation
+
+Everything you need to know to setup the project locally can be found in our [documentation](DOCUMENTATION.md). Please follow our [coding conventions](DOCUMENTATION.md/#coding-conventions).
+
+## Reporting Bugs and Issues
+
+If you think you've found a bug or accessibility issue, feel free to open an issue here on GitHub. Before opening a new issue, please answer this checklist for yourself:
+
+1. I ensured that the problem is not caused by my local environment (e.g., a browser plugin, a really old browser version or unusual browser settings)
+2. I checked that no GitHub issue was already filed for this bug
+
+If you answered everything with yes, go ahead and [create a new GitHub issue](https://github.com/a11yphant/a11yphant/issues/new/choose). Be sure to include as much information as possible so we can reproduce the bug.
+
+## Feature Requests
+
+Feature requests are very welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince us of this feature. Please provide as much detail and context as possible.
+
+As we are only a small team of developers and designers, you are invited to contribute by creating a pull request. Please discuss your proposed changes with us before starting implementation. Otherwise, they might collide with our plans. Also, remember to follow our [coding conventions](DOCUMENTATION.md/#coding-conventions). 
```

Comment:
I would not encourage people to create pull requests by themselves. At least they should discuss the intended change with us before starting to work on it. This will prevent disappointment when someone implements a new feature and we decide that we do not wan't to add it to the platform.